### PR TITLE
Added Projects

### PR DESCRIPTION
**Pull request by GiupiDeLuca**
I added the following projects: 
Mimo, UniFi, ZoomSwap, MinMax Finance, Parrot Defi, Magic Land, WOWswap, Cyclone, StarCrazy

### CODE REVIEW

File: data/ecosystems/c/cyclone.toml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Ecosystem Level Information
+title = "Cyclone"
```
Comment (eherrerosj) [inline data/ecosystems/c/cyclone.toml:2]: should also be added as a `sub_ecosystem` under IoTeX

File: data/ecosystems/m/magiclandfinance.toml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Ecosystem Level Information
+title = "Magic Land"
```
Comment (eherrerosj) [inline data/ecosystems/m/magiclandfinance.toml:2]: should also be added as a `sub_ecosystem` under IoTeX

File: data/ecosystems/p/parrotdefi.toml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Ecosystem Level Information
+title = "Parrot DeFi"
```
Comment (eherrerosj) [inline data/ecosystems/p/parrotdefi.toml:2]: should also be added as a `sub_ecosystem` under IoTeX

File: data/ecosystems/w/wowswap.toml [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Ecosystem Level Information
+title = "WOWswap"
```
Comment (eherrerosj) [inline data/ecosystems/w/wowswap.toml:2]: should also be added as a `sub_ecosystem` under IoTeX

File: data/ecosystems/z/zoomswap.toml [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Ecosystem Level Information
+title = "ZoomSwap"
```
Comment (eherrerosj) [inline data/ecosystems/z/zoomswap.toml:2]: should also be added as a `sub_ecosystem` under IoTeX

**Review by eherrerosj — CHANGES_REQUESTED** [review]
Thanks for the PR @GiupiDeLuca! Left a few comments. Mainly 3 changes are needed:
1. adding all these projects under IoTeX toml so that it is seen as a parent 
2. removing duplicates
3. fixing one of the project titles

**Comment by GiupiDeLuca** [comment]
Thanks for the feedback @eherrerosj - I think I should have everything now. 
- I deleted the duplicates
- Added the projects under iotex
- Fixed the wrong title

**Comment by GiupiDeLuca** [comment]
@eherrerosj I have 2 checks failing, would you be able to help with those? Not sure what I did wrong. 😆

**Comment by eherrerosj** [comment]
> @eherrerosj I have 2 checks failing, would you be able to help with those? Not sure what I did wrong. 😆

Thanks for the update. Not sure by 97fe is failing (can ignore), but f881It is failing because it's now a duplicate (seems like we have it already in our db), you can remove that addition then

**Comment by eherrerosj** [comment]
yo can also bring the changes from your other PR to this one

**Comment by GiupiDeLuca** [comment]
@eherrerosj I removed f881, I see there is minimax-finance, but it seems to be a different project than the MinMax Finance I added. Anyway, I also removed MinMax from the list of iotex sub_ecosystems for now, maybe later on we can check if these two are in fact different project with an almost identical name, or they are indeed the same. Thanks again for the feedback

**Comment by eherrerosj** [comment]
thank you @GiupiDeLuca !

**Pull request merged**